### ISSUE TITLE
Only Shows First Artist Played, no other Rich Pressence information

### ISSUE THREAD

**Issue by nothingmachine** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#103]
With Tidal and TDRP running, I can select Tidal from the list when no song is playing, but it doesn't update as songs or artists change. When selecting the app while a song is playing, it only shows the artists name and never updates. 
![image](https://github.com/user-attachments/assets/590eaaae-d77d-421d-a153-27e8a6ce7f9a)
![Screenshot 2024-07-17 120445](https://github.com/user-attachments/assets/494fd3d4-77e2-47d9-8517-1a493b732a79)

**Comment by purpl3F0x** [comment]
If i recall correctly you don't have to select anything form discord. 
The instructions are outdated 🥲

**Comment by nothingmachine** [comment]
Nothing happens at all if I just run the applications unfortunately.

**Comment by purpl3F0x** [comment]
Does the app report that something is playing, when you click the taskbar icon ?

**Comment by nothingmachine** [comment]
Yes it looks like the app in the toolbar is connected to tidal and discord: 
![image](https://github.com/user-attachments/assets/2966e6d6-f812-49db-a7e1-5528a6f0b10c)

**Comment by purpl3F0x** [comment]
I'm not sure, what's happening. Try exiting Discord and reopening it.

**Comment by SuperBravo45** [comment]
> I'm not sure, what's happening. Try exiting Discord and reopening it.

FYI, I'm having the exact same issue upon trying the app now

**Comment by purpl3F0x** [comment]
Does the issue persist?

**Comment by nothingmachine** [comment]
Truthfully I have not tried the app again since I reported the issue in July as I couldn't get it to work.

**Comment by SuperBravo45** [comment]
> Does the issue persist?

Seems to be working for me now, actually.

**Comment by derpina-ai** [comment]
> With Tidal and TDRP running, I can select Tidal from the list when no song is playing, but it doesn't update as songs or artists change. When selecting the app while a song is playing, it only shows the artists name and never updates. 

still happens on my end. no clue how to fix it. restarted everything 3x over.

Update: got it working!! The only way to get it working is pause EVERYTHING, make sure to add TIDAL in discord, not a song name as a game activity. It will update when you start playing in Tidal.
